pyproject.toml

```
@@ -23,13 +23,11 @@ dependencies = [
     "photutils >=1.13.0",
     "pyparsing >=2.4.7",
     "requests >=2.26",
-    # "rad>=0.21.0,<0.22.0",
-    "rad @ git+https://github.com/spacetelescope/rad.git",
-    # "roman_datamodels>=0.21.0,<0.22.0",
-    "roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git",
+    "roman_datamodels>=0.22.0,<0.23.0",
+    # "roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git",
     "scipy >=1.11",
-#    "stcal>=1.8.0,<1.9.0",
-    "stcal @ git+https://github.com/spacetelescope/stcal.git@main",
+   "stcal>=1.9.0,<1.10.0",
```

```suggestion
   "stcal>=1.10.0,<1.11.0",
```
Look like we need a stcal release also to support https://github.com/spacetelescope/romancal/pull/1395

Would you make a 1.10.0 release for stcal?

---

on it!